Print() scrolling text

LiamHowell opened the pull request:
Added print functions as suggested in [User-experience improvements #2](https://github.com/CoreElectronics/CE-PiicoDev-SSD1306-MicroPython-Module/issues/2), along with an example tested on a Raspberry Pi Pico.

All arguments for the print() function are documented, the other supporting functions remain undocumented in the README.

Cheers,
Liam.

Commit:
Added some functions to support scrolling text

Commit:
Completed functions

Added print(), trunctate_text(), draw_print_text(), _init_print_text_func() along with print.py in examples

CoreProduction commented:
I'm taking this for a test drive! The expected scrolling behaviour is really nice! 
I have taken the `print.py` example and reduced it into the following. I expected the text to appear part-way up the screen, but it still appears at the bottom line, and scrolls up. 

```from PiicoDev_SSD1306 import *
from PiicoDev_Unified import sleep_ms

display = create_PiicoDev_SSD1306()
display.fill(0)

display.print("insert 1", line_num = 3)
display.print("insert 2", line_num = 4)
```

produces the result
```shell
[blank lines above]
insert 1
insert 2
```

Is the `line_num` feature still under heavy development? Or did I find a bug?

Commit:
Fixed line_num, improved text colour

Added a property to get the display size, text colour is stored in disp_lst and temp_lst,, font size is stored in the library for other functions to access, general cleaning

LiamHowell committed:
Update README.md

LiamHowell committed:
Update README.md

LiamHowell commented:
Hi Michael,

Good catch, a bit of both, the argument `auto_scroll` was required to insert lines at specific locations which didn't make too much sense.
Thanks for the feedback! I've fixed the issue and added more functionality to allow Makers to create menus with an included example.

https://github.com/LiamHowell/CE-PiicoDev-SSD1306-MicroPython-Module/blob/print()-scrolling-text/example/print_menu.py

Also included is a way to access the display's height and width from a property
